ServerCore/Startup.cs

```diff
@@ -28,28 +40,41 @@ public void ConfigureServices(IServiceCollection services)
                     options.Conventions.AuthorizeFolder("/Teams");
                 });
 
-            services.AddDbContext<PuzzleServerContext>
-                (options => options.UseLazyLoadingProxies()
-                    .UseSqlServer(Configuration.GetConnectionString("PuzzleServerContext")));
+            // Use SQL Database if in Azure, otherwise, use localdb
+            if (Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development")
+            {
+                services.AddDbContext<PuzzleServerContext>
+                    (options => options.UseLazyLoadingProxies()
+                        .UseSqlServer(Configuration.GetConnectionString("PuzzleServerSQLConnectionString")));
+            }
+            else
+            {
+                services.AddDbContext<PuzzleServerContext>
+                    (options => options.UseLazyLoadingProxies()
+                        .UseSqlServer(Configuration.GetConnectionString("PuzzleServerContextLocal")));
+            }
 
             services.AddAuthentication().AddMicrosoftAccount(microsoftOptions =>
             {
-                microsoftOptions.ClientId = Configuration["Authentication:Microsoft:ApplicationId"];
-                microsoftOptions.ClientSecret = Configuration["Authentication:Microsoft:Password"];
+                microsoftOptions.ClientId = Configuration["Authentication-Microsoft-ApplicationId"];
+                microsoftOptions.ClientSecret = Configuration["Authentication-Microsoft-Password"];
+
             });
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
-            if (env.IsDevelopment())
+            PuzzleServerContext.UpdateDatabase(app);
+
+            if (env.IsDevelopment() || Environment.GetEnvironmentVariable("ASPNETCORE_ENVIRONMENT") == "Development")
             {
                 app.UseBrowserLink();
                 app.UseDeveloperExceptionPage();
-                PuzzleServerContext.UpdateDatabase(app);
             }
             else
             {
+                app.UseDeveloperExceptionPage();
```

Comment:
Please remove this or file an issue to do so

Reply:
Removed (was for debugging)